### PR TITLE
GitHub community health files

### PR DESCRIPTION
**Pull request by niden**
Created .github folder for common issue templates and funding
Added stale.yml for the stale bot

### CODE REVIEW

**Comment by danhunsaker** [comment]
Any particular reason to move the files into a subdir? They work as well in the root... Is there another use for this repo that you had in mind?

**Comment by niden** [comment]
@danhunsaker I was just following what we did in phalcon. so the fiiles that we had in .gitignore I put them in .gitignore.

Not sure if it works when you have them all in the root. If it does I will re-move them.

**Comment by niden** [comment]
@danhunsaker I checked the docs. The files do not need to be in their own `.github` folder so I moved them all to the root.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+> Questions should go to https://discord.gg/MCDdHfy
+
+### Expected and Actual Behavior
+
+> **Describe what you are trying to achieve and what goes wrong.**
+
+> Provide output if related. 
+
+```php
```
Comment (danhunsaker) [inline ISSUE_TEMPLATE.md]: Probably want to remove highlighting on this one...

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+// paste output here
+```
+
+> Provide minimal script to reproduce the issue
+
+```php
```
Comment (danhunsaker) [inline ISSUE_TEMPLATE.md]: ...and switch this one to `bash`...

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+Hello!
+
+I am issuing this Pull Request to make Microbox even more awesome!
+
+* Type: bug fix | new feature | code quality | documentation
+* Link to issue:
+
+**In raising this pull request, I confirm the following:**
+
+- [ ] I have read and understood the [Contributing Guidelines](https://github.com/mu-box/core/blob/master/CONTRIBUTING.md)
```
Comment (danhunsaker) [inline PULL_REQUEST_TEMPLATE.md:10]: We don't actually have one of these... Can we add one to this PR?

File: stale.yml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: true
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  Thank you for contributing to this issue. As it has been 90 days since the last activity, we are automatically closing the issue. This is often because the request was already solved in some way and it just wasn't updated or it's no longer applicable. If that's not the case, please feel free to either reopen this issue or open a new one. We will be more than happy to look at it again! You can read more here: https://blog.phalcon.io/post/github-closing-old-issues
```
Comment (danhunsaker) [inline stale.yml]: May want to drop the "read more at" bit, since we're not Phalcon?

File: stale.yml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+# Comment to post when removing the stale label.
+# unmarkComment: >
+#   Your comment here.
+
+# Comment to post when closing a stale Issue or Pull Request.
+#closeComment: >
+#  Thank you for contributing to this issue. As it has been 90 days since the last activity, we are automatically closing the issue. This is often because the request was already solved in some way and it just wasn't updated or it's no longer applicable. If that's not the case, please feel free to either reopen this issue or open a new one. We will be more than happy to look at it again! You can read more here: https://blog.phalcon.io/post/github-closing-old-issues
```
Comment (danhunsaker) [inline stale.yml]: Same here, even though it's commented.

**Review by danhunsaker — CHANGES_REQUESTED** [review]
Left some thoughts inline.

**Comment by niden** [comment]
@danhunsaker updated

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+## Contributions
+
+Contributions to Microbox should be in the form of GitHub pull requests. Each pull request will be reviewed by a core contributor (someone with permission to merge). The pull request can then be merged, feedback can be added, changes requested or the pull request can be denied. All contributions should follow this format, even those from core contributors.
+
+## Questions & Support
+
+We use the GitHub issues for tracking bugs and feature requests and have limited bandwidth to address all of them. Thus we only accept bug reports, new feature requests and pull requests in GitHub. Our great community and contributors are happy to help you though! Please use these community resources for obtaining help.
+
+* Come join the Microbos [Discord](https://discord.gg/MCDdHfy)
```
Comment (danhunsaker) [inline CONTRIBUTING.md]: Typo!

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+* If you still believe that what you found is a bug, please [open an issue](https://github.com/mu-box/core)
+
+Please report bugs when you've exhausted all of the above options.
+
+## Bug Report Checklist
+
+* Make sure you are using the latest released version of Microbox before submitting a bug report. Bugs in versions older than the latest released one will not be addressed by the core team (so long as the version is not LTS).
+* If you have found a bug it is important to add relevant information to reproduce it. This way the core team can find and fix it quicker.
+* Please ensure that information such as OS and Microbox versionare part of the bug report
```
Comment (danhunsaker) [inline CONTRIBUTING.md]: Lost a space in here...

**Review by danhunsaker — CHANGES_REQUESTED** [review]
A couple of typos, then this is good to go!

**Comment by niden** [comment]
@danhunsaker Fixed

**Review by danhunsaker — APPROVED** [review]

**Comment by danhunsaker** [comment]
Thanks!

**Pull request merged**